config/initializers/simple_form.rb

```diff
@@ -0,0 +1,190 @@
+# Use this setup block to configure all options available in SimpleForm.
```

Comment:
these files are in another PR, but I merged that branch in so I could streamline forms above